### PR TITLE
update intent and basedOn fields, add tests

### PR DESCRIPTION
**Pull request by dan-jones-iw**

### CODE REVIEW

File: coordinator/tests/services/translation/response/release/release-response.spec.ts [hunk 1]
```diff
@@ -9,19 +8,23 @@ import * as fs from "fs"
...
-  getPractitioners
+  getPractitioners,
+  getProvenances
 } from "../../../../../src/services/translation/common/getResourcesOfType"
 import {getRequester, getResponsiblePractitioner} from "../common.spec"
 
 describe("outer bundle", () => {
   const result = createOuterBundle(getExamplePrescriptionReleaseResponse())
-  console.log(LosslessJson.stringify(result))
+  LosslessJson.stringify(result)
```
Comment (mfjarvis) [inline coordinator/tests/services/translation/response/release/release-response.spec.ts:27]: Don't think this call is doing anything

Comment (mfjarvis) [reply coordinator/tests/services/translation/response/release/release-response.spec.ts:27]: If it's there to check that the result can be stringified, I'd use:
```
expect(() => LosslessJson.stringify(result)).not.toThrow()
```

Comment (dan-jones-iw) [reply coordinator/tests/services/translation/response/release/release-response.spec.ts:27]: yea not really sure why its there in the first place
i just removed all the console logs to stop tests being so annoying
think ill remove all the stringifies except the top level to ensure everything can be stringified.

File: coordinator/src/services/translation/response/release/release-medication-request.ts [hunk 2]
```diff
@@ -40,8 +46,8 @@ export function createMedicationRequest(
       createItemNumberIdentifier(lineItem.id._attributes.root)
     ],
     status: getStatus(lineItem.pertinentInformation4.pertinentItemStatus),
-    basedOn: fhir.createReference(releaseRequestId.toLowerCase()),
-    intent: fhir.MedicationRequestIntent.REFLEX_ORDER,
+    basedOn: fhir.createReference(lineItem.id._attributes.root),
```
[the remark this answers is not in the document]
Comment (mfjarvis) [reply coordinator/src/services/translation/response/release/release-medication-request.ts:49]: Think you added the `toLowerCase()` to the wrong field. The `identifier` already gets lower cased inside `createItemNumberIdentifier()`. I was talking about `basedOn`

**Review by callumtipper-IW — APPROVED** [review]

**Pull request merged**